src/main/java/com/epam/javacourses2016/task13/AbstractGraphCreator.java

```diff
@@ -11,7 +11,10 @@
      * @return Граф указанной конфигурации.
      */
     public static AbstractGraph createGraph(int numberNodes){
-        return null;
+        if(numberNodes <= 0) {
```

Comment:
java way is to throw on predicates
Or if null can be returned, `Optional` should be used as returned value